src/main/java/edu/ksu/canvas/requestOptions/UnEnrollOptions.java

```diff
@@ -0,0 +1,24 @@
+package edu.ksu.canvas.requestOptions;
```

Comment:
Hmm this is a slight oddball. The other classes in this package wrap functionality to add multiple optional parameters to a request. This is just a straight up enum. But I'm not sure if there is a better place to put it either. So while it seems a little odd, I don't really have a suggested change here.

Reply:
Yes, I noticed that too. Guess we can start with this and refactor down the road? Open to suggestions.

Reply:
Thanks ToeBee, looking now

Reply:
Pushed the javadoc fixes, let me know how it goes!